applications/nrf5340_audio/broadcast_sink/Kconfig.defaults

```diff
@@ -50,6 +50,11 @@ config BT_PAC_SNK
 config BT_AUDIO_RX
 	default y
 
+config BT_PER_ADV_SYNC_TRANSFER_RECEIVER
+	default y
+
+config BT_DEVICE_NAME
```

Comment:
Since the user could build the code for broadcast sink and source at the same time.
And the default setting will make sink/source use the same BT_DEVICE_NAME is we don't assign the specific name for sink.